### ISSUE TITLE
/config_list/ API always returns archived configs

### ISSUE THREAD

**Issue by pspacek** [isc-projects/perflab#15]
API endpoint `/api/config_list/` always returns configs with `archived: true` property, even when called as `/api/config_list/?archived=false`.

Version: 2e447fb

**Comment by raybellis** [comment]
That's the documented behaviour - filtering on the `archived` flag currently happens client side.    If you want the ability to to filter server side then that's a new feature request.

**Comment by pspacek** [comment]
Huh, I'm confused. Docs https://github.com/isc-projects/perflab/blob/master/doc/api.md#get-apiconfig_list say:
```
Get an array of all active configurations in abbreviated form. Optionally takes a query parameter of ?archived=true to also include archived configurations.
```
Please note this is in section `REST API`.

If we agree that current behavior is correct I will prepare MR to remove this parameter from API docs. Do you agree with that?

**Comment by raybellis** [comment]
You're right - the code did previously exist, but was removed in preference for client side filtering, without updating the docs.   Fixed with 4c7b913.
